### ISSUE TITLE
Firmware not connecting on new Kobra Max Motherboard (new Mobo revision)

### ISSUE THREAD

**Issue by DTqutHDR** [ryanpie86/Anycubic-Kobra-Max-Klipper-Conversion#1]
New motherboard from Anycubic (Trigorilla_Pro_B_V1.0.2) already has the R65->R66 mod done.

However following these steps I was not able to connect to the motherboard, which previously worked on the revision A board. I'm wondering if a new firmware build for the new revision of the mobo needs to be done.

**Comment by ryanpie86** [comment]
Yes, the B revision has different stepper drivers among other issues.
Unfortunately I don't have one to play with. If you happen to figure it
out, feel free to submit it here..

**Comment by wuhb999** [comment]
> New motherboard from Anycubic (Trigorilla_Pro_B_V1.0.2) already has the R65->R66 mod done.
> 
> However following these steps I was not able to connect to the motherboard, which previously worked on the revision A board. I'm wondering if a new firmware build for the new revision of the mobo needs to be done.

Has this problem been solved?

**Comment by wuhb999** [comment]
> New motherboard from Anycubic (Trigorilla_Pro_B_V1.0.2) already has the R65->R66 mod done.
> 
> However following these steps I was not able to connect to the motherboard, which previously worked on the revision A board. I'm wondering if a new firmware build for the new revision of the mobo needs to be done.

![mmexport1703245046091.png](https://github.com/ryanpie86/Anycubic-Kobra-Max-Klipper-Conversion/assets/153477003/382cefb5-a8af-49bc-b191-d801b8ee85a7)

You can press this image settings
